test/test_cmd.py

```diff
@@ -0,0 +1,33 @@
+""" Tests for the command line executable. """
+import os
+import unittest
+
+from edx_lint.cmd import main
+
+
+class WriteCommandTest(unittest.TestCase):
+    """ Tests for the write command. """
+
+    def call_command(self, argv=None):
+        """ Call an edx_lint script command.
+
+        Arguments:
+            argv (list) -- arguments to pass to the edx_lint script
+        """
+        return main.main(argv)
+
+    @unittest.skip('Default test runners have issues finding the file.')
```

Comment:
Note that I've added a test that is broken for everything but PyCharm. Still trying to figure out why.